src/components/Requirements/SubRequirement.vue

```diff
@@ -255,7 +262,11 @@ export default Vue.extend({
         if (!this.isCompleted) {
           slots.push({
             isCompleted: false,
-            courses: generateSubReqIncompleteCourses(allTakenCourseIds, subReqEligibleCourses[0]),
+            courses: generateSubReqIncompleteCourses(
+              allTakenCourseIds,
+              subReqEligibleCourses[0],
+              this.subReq.requirement.id
```

Comment:
Same here. I think this may be of type `number` not `string`. I believe `requirementID` was defined as having type `string` for `AppFirestoreSemesterCourseWithRequirementID`